README.md

```diff
@@ -316,8 +320,58 @@ The Realm Dart package is `realm_dart`
 
     _*This file should be committed to source control_
 
-* For more usage of Realm Dart see the Realm Flutter usage above.
+* For more information on using the Realm Dart Standalone SDK see the Realm Flutter SDK usage above.
```

Comment:
```suggestion
* For more information on using the Realm Dart Standalone see the Realm Flutter usage above.
```
as mentioned above we don't need to emphasize the word SDK each time

Reply:
OK